cluster/charts/rook-nfs/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for rook-nfs-operator
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  prefix: rook
+  repository: rook/nfs
+  tag: %%VERSION%%
```

Comment:
Let's use `tag: VERSION` (without the `%`). We just went back and forth on the ceph chart and that's what we ended up with.